internal/controllers/extension_controller.go

```diff
@@ -203,11 +209,15 @@ func (r *ExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alpha1.Ext
 		// originally Reason: ocv1alpha1.ReasonInstallationStatusUnknown
 		ext.Status.InstalledBundleResource = ""
 		setInstalledStatusConditionUnknown(&ext.Status.Conditions, err.Error(), ext.GetGeneration())
+		setProgressingStatusConditionFalse(&ext.Status.Conditions, "app install failed", ext.GetGeneration())
```

Comment:
Same comment here as above. A rule of thumb is: if we're going to end up returning an error from `Reconcile`, that means we're going to retry, which means we're still progressing.